packages/paste-website/src/components/site-wrapper/site-header/SiteHeaderDesktop.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+import {Stack} from '@twilio-paste/stack';
+import {GithubLink} from './GithubLink';
+import {SiteHeaderDesktopSearch} from './SiteHeaderDesktopSearch';
+import {SiteHeaderLogo} from './SiteHeaderLogo';
+import {ThemeSwitcher} from '../../ThemeSwitcher';
+import {useNavigationContext} from '../../../context/NavigationContext';
+
+interface SiteHeaderDesktopProps {
+  searchValue: string;
+  onSearchChange: (newSearchValue: string) => void;
+}
+
+const SiteHeaderDesktop: React.FC<SiteHeaderDesktopProps> = ({searchValue, onSearchChange}) => {
+  const {pathname} = useNavigationContext();
+  const isHomepage = pathname === '/' || pathname === '/new';
```

Comment:
I'm seeing the theme switcher on /new. https://share.getcloudapp.com/WnulR0dm
Might need to change this to:

```suggestion
  const isHomepage = pathname === '/' || pathname === '/new/';
```

Reply:
Oh right, because both `/new` and `/new/` are valid URLs. I need to check for both. Good catch!